cluster-autoscaler/version/version.go

```diff
@@ -17,4 +17,4 @@ limitations under the License.
 package version
 
 // ClusterAutoscalerVersion contains version of CA.
-const ClusterAutoscalerVersion = "1.26.4"
+const ClusterAutoscalerVersion = "1.26.11"
```

Comment:
This is a CA version not a K8s version. We should bump the patch number once to `1.26.5`

Reply:
This was autogenerated. Overwritten now.